magento-research/pwa-studio
# bug: Fix peregrine watcher and watch:all

## Description

The peregrine watcher was broken! Specifically the watch process was just running `yarn run build --watch` which passes `--watch` to the build command. The problem there was that `yarn build` uses `concurrently` which does not allow arguments to the concurrent processes. For example, the following is not allowd:

`concurrently process_one --watch process_two --watch`.

Instead, concurrently suggests we write scripts that have arguments pre-coded that are used within concurrently. Thus this change.

## Related Issue





Closes #1314.

## Verification Steps

1. From the root, run `yarn run watch:all`, open the store and make a change to a peregrine file like `useToasts`. The page should reload!
2. From the root, run `yarn run watch:peregrine`. Open `useToasts` in `peregrine/esm`. Make a change to the `useToasts` in `src` and see that the `esm` file updates.

## Screenshots / Screen Captures (if appropriate)

## Proposed Labels for Change Type/Package




## Checklist:
